core/blockly.js

```diff
@@ -294,6 +294,9 @@ Blockly.onMouseUp_ = function(e) {
  * @private
  */
 Blockly.onMouseMove_ = function(e) {
+  if (event.touches && event.touches.length >= 2) {
```

Comment:
Wait, 'event'?  Shouldn't that be 'e'?  Oddly, this worked in testing, then failed later.